cmd/webpackager/flag_packager.go

```diff
@@ -50,7 +52,10 @@ var (
 	flagPrivateKey   = flag.String("private_key", "", `Private key PEM file. (required)`)
 
 	// Processor
-	flagSizeLimit = flag.String("size_limit", "4194304", fmt.Sprintf(`Maximum size of resources in bytes allowed for signed exchanges, or %q to set no limit.`, noSizeLimitString))
+	flagSizeLimit = flag.String("size_limit", "4194304", `Maximum size of resources in bytes allowed for signed exchanges, or "none" to set no limit.`)
```

Comment:
I changed my mind and stopped using `fmt.Sprintf` here.

The description of `--expiry` contains `"168h"`, which should match `maxExpiry`. `fmt.Sprintf` doesn't format a Duration in a favorable way: `maxExpiry.String()` gives `"168h0m0s"`, which I don't like (because it looks like we always have to include `"XXmXXs"`, not simply `"XXXh"`).

Now we have at least one flag we don't use `fmt.Sprintf` with, I think it's simpler to avoid `fmt.Sprintf` for all flags.

Reply:
Make sense. Thanks for the explanation.